doc/whats_new.rst

```diff
@@ -36,6 +36,10 @@ Changelog
 - Add feature to convert param and json files to HDF5 format, by `George Dang`_
   in :gh:`723`
 
+- Fixed bug in :func:`~hnn_core/network/pick_connection` where connections were
+  returned for cases when there should be no valid matches, by `George Dang`_
+  in :gh:`739`
```

Comment:
This is perhaps the nitpickiest of nitpicks, but I just noticed we haven't been consistent about adding or omitting a period at the end of each entry. We should decide on a convention and stick to it.

Reply:
Also I used past tense where where the others are present... I'll address that and add periods to everything else.

Reply:
Actually I'll leave the periods to another PR.